packages/core/src/request.ts

```diff
@@ -172,6 +184,22 @@ export class Request<UserData extends Dictionary = Dictionary> {
         this.headers = { ...headers };
         this.handledAt = handledAt as unknown instanceof Date ? (handledAt as Date).toISOString() : handledAt!;
 
+        this.requestHandlerTimeoutSecs = requestHandlerTimeoutSecs;
+        this.requestTimeoutSecs = requestTimeoutSecs;
+
+        Object.defineProperty(
+            this,
+            'requestHandlerTimeoutMillis',
+            // If zero was provided, this will still return null.
+            { enumerable: false, value: requestHandlerTimeoutSecs ? requestHandlerTimeoutSecs * 1e3 : null },
+        );
+
+        Object.defineProperty(
+            this,
+            'requestTimeoutMillis',
+            { enumerable: false, value: requestTimeoutSecs ? requestTimeoutSecs * 1e3 : null },
+        );
```

Comment:
Is this actually required?

Reply:
Well, do we want these properties to be publicly viewable? Or is underscoring enough.

Reply:
No, I mean do we need this at all?

Reply:
These properties?

Reply:
They're nice-to-haves, but we don't necessarily need them.

Reply:
The alternative is ugly ternary operators in certain places.

Reply:
I wouldn't be so sure :) https://github.com/apify/crawlee/pull/1560#discussion_r977078452